src/org/usfirst/frc/team694/robot/RobotMap.java

```diff
@@ -47,7 +47,11 @@
     /***************************************************************************
      * Lift Constants
      *************************************************************************/
-    public static final double LIFT_ENCODER_RAW_MULTIPLIER = -1;
+    public static final double DIAMETER_OF_ENCODER_SPROCKET = 2.873; 
+    public static final double TOTAL_CARRIAGE_MOVEMENT = 95.25;
+    public static final double LIFT_ENCODER_RAW_MULTIPLIER = DIAMETER_OF_ENCODER_SPROCKET * Math.PI * 3 / 1024;
+    public static final double MIN_HEIGHT_OF_GRABBER = 23;
+    public static final double MAX_HEIGHT_OF_LIFT = TOTAL_CARRIAGE_MOVEMENT + MIN_HEIGHT_OF_GRABBER;
```

Comment:
Maybe this isn't an issue, but I'd consider how this is used in our code.

If we're resetting our encoders to 0 upon hitting the limit switch, then we start at "0" and not at "MIN_HEIGHT_OF_GRABBER".